README.md
hrbrmstr's gists

- [No description provided](86fb5284178f1a87248929bb719a2bd4) @ 2018-06-09T22:08:22Z
- [No description provided](725342cf2bcb83a03427df3cd0e8ad53) @ 2018-06-07T15:26:09Z
- [No description provided](d07b18ff928a875152d3911e3ea37d20) @ 2018-06-06T10:25:46Z
- [No description provided](c07ddf290369d954a54443b6b09faae3) @ 2018-06-01T17:48:27Z
- [No description provided](9a3233cc96ee4334821fc76b28f4d33c) @ 2018-05-31T01:45:54Z
- [No description provided](58b827dea95a7adf0f0e751b44b75b0c) @ 2018-05-29T11:55:28Z
- [No description provided](4d16528700312cd25cc1f79bb1aa2fc6) @ 2018-05-28T11:36:01Z
- [Ref: https://twitter.com/omespino/status/1000524541829971968](4232f38b5ff172dd06acc093ccfe98eb) @ 2018-05-27T13:48:52Z
- [A JSON file of sources, headlines and (where existing) intro snippets for the "Amazon Alexa 'Scandal'"](1423cba2aa1bdb9361066a7af21f9a1e) @ 2018-05-25T11:16:53Z
- [No description provided](0be255289531061e5ef01553c4409324) @ 2018-05-19T11:34:19Z
- [No description provided](85f4fb237f05c659c64713f4f1242562) @ 2018-05-15T19:29:23Z
- [No description provided](ae671ff9668dbf64ca85f33a72450a25) @ 2018-05-13T11:59:42Z
- [No description provided](5b7d9eac47f24c33ca8c3c1f8ea7134b) @ 2018-05-09T16:13:36Z
- [https://twitter.com/_odddie_/status/993880501197864960 thread capture](c0b2bc6e7a90b401f46f39f2f34d41c7) @ 2018-05-08T16:19:54Z
- [No description provided](62d2346371e6cfca4bb7f48deb87bc62) @ 2018-05-03T15:50:06Z
- [No description provided](b5ac4d64c19ea26cb18df743241ad96c) @ 2018-04-30T18:41:06Z
- [No description provided](9a323a7fdbc96dae3b3f7b26a481bbc1) @ 2018-04-30T01:29:44Z
- [No description provided](09a255ddc403d0234e018d4698693cfe) @ 2018-04-28T12:29:38Z
- [No description provided](ff76bc29134e45b8b8b960c278d89aa1) @ 2018-04-27T17:21:06Z
- [Text of Jessica Payne's Twitter thread: https://twitter.com/jepayneMSFT/status/988895496381984770](c31fded255c85672cbeb804bb3131b3b) @ 2018-04-25T11:35:56Z
- [No description provided](4ada681bd4ddfcc707b2824c7026070e) @ 2018-04-23T22:20:54Z
- [No description provided](92c982751c9cb60a5ed358e3f554cd2c) @ 2018-04-23T13:26:33Z
- [No description provided](ce2c49c7d4b7027437728192e278781c) @ 2018-04-20T18:15:10Z
- [No description provided](f244a72633435484f3f0017855c1cc60) @ 2018-04-18T16:07:16Z
- [No description provided](715a18e14ae926e1776ae74f33517e13) @ 2018-04-15T20:05:47Z
- [No description provided](bf034913027f3447325713223c4d5f79) @ 2018-04-13T15:21:38Z
- [No description provided](e54415b033bb194f4eb3e639b3d8d76c) @ 2018-04-10T20:44:41Z
- [No description provided](33d29bb39eaa7f2f1e95308038f85b59) @ 2018-04-08T12:26:42Z
- [Boyer-Moore Implementations for @coolbutuseless' comparisons](05c6b05efab799eb4522aa1ff7d80a37) @ 2018-04-04T13:24:29Z
- [No description provided](276072b0a83ee25328aaa1c046880ce8) @ 2018-04-04T01:08:01Z
- [user-settings.js](fe406ff80e22b45895546853e29b2651) @ 2018-03-13T19:40:12Z
- [No description provided](42aff98aca45d29f4cda5e731bacdd8d) @ 2018-03-13T15:18:38Z
- [Self-executing single-file Java programs...](9232c8f5b191c6601b04298b5399b96f) @ 2018-03-02T22:12:34Z
- [No description provided](a3b9a00d4cfb3aae2aad8d8f9d325dbe) @ 2018-02-26T17:15:48Z
- [Getting rpy2 to work on macOS 10.12 / 10.13 with pip3 and gcc](0631d9a651f4a40378ddb9ea691a1356) @ 2018-02-26T12:37:24Z
- [No description provided](7dd469e8ed69945e3f7f0c53cc1bdd14) @ 2018-02-10T22:40:09Z
- [No description provided](f54605c27c7b102c40e2a1412fa98502) @ 2018-02-10T19:26:44Z
- [No description provided](c1be4e9f424a77c80595174678ab74e1) @ 2018-02-04T18:16:49Z
- [faster IPv4 Hilbert curves in R + ggplot2](a244a4fdd65f7c987f3de2d08e0ceb9a) @ 2018-02-04T02:58:26Z
- [No description provided](9542742fb6183462428894c66836f50d) @ 2018-01-25T18:55:08Z
- [databot firehose](8a49b78f6567cb3de0d074a82a9b3327) @ 2018-01-24T02:54:13Z
- [No description provided](157d2e526fa509f1593b182c5b054bf0) @ 2018-01-22T17:40:30Z
- [No description provided](82eae64f0fdd9e4eae7d7b9443d6b3f8) @ 2018-01-21T13:51:38Z
- [https://rud.is/b/2018/01/18/bitcoin-world-map-bubbles/](0303ba204e3f56def4daa261db340135) @ 2018-01-18T14:18:48Z
- [No description provided](bb93a8f2b378c6112fedbe581690bf21) @ 2018-01-16T13:48:51Z
- [This function will walk you through the creation of a remote repository on github without wasting your precious time going to the website :)](699f225cce4f37dcd1a78b3ecec3b6fe) @ 2018-01-16T04:59:28Z
- [No description provided](8025f9c18a0fb75279083747fab5d3f3) @ 2018-01-12T15:30:26Z
- [non-selenium alternative to http://www.masalmon.eu/2018/01/07/rainbowing/](4cabe4af87bd2c5fe664b0b44a574366) @ 2018-01-07T14:43:59Z
- [Connect R (#rstats) to heroku PostgreSQL — https://www.heroku.com/postgres](45c67103a9728f59212cd13262adca74) @ 2018-01-06T09:21:02Z
- [No description provided](122e9a3b91711cea1e222b321416043b) @ 2018-01-05T12:04:24Z
- [No description provided](34964dc9320ca3cb30063be95b992128) @ 2018-01-04T21:02:42Z
- [No description provided](a47e74ef707d7ed3735ed59a92a01125) @ 2018-01-02T13:43:43Z
- [No description provided](ac802de3b98299a73027a79ed91bfe4d) @ 2018-01-02T01:32:02Z
- [No description provided](6c4330efad178a572655800f4f713764) @ 2018-01-02T01:20:47Z
- [No description provided](27cf4e6d3fc34da84b4006a6d209ba9e) @ 2017-12-31T16:21:49Z
- [No description provided](d033831a257d8dd4edfde7a06f14e573) @ 2017-12-30T13:06:36Z
- [No description provided](247f4353f10f2acc9d0ef417a479a442) @ 2017-12-28T12:55:00Z
- [No description provided](28d4a79ca778e019b7c6ae8d16980dd5) @ 2017-12-27T18:53:54Z
- [No description provided](470890ad2e0319f5d226eb2b5b4ef2c5) @ 2017-12-27T16:10:46Z
- [No description provided](cf726437951068992ea4ef14f6c2d892) @ 2017-12-22T15:57:11Z
- [No description provided](5ae822585ab16e7b2b08394b4151d8d0) @ 2017-12-14T14:24:26Z
- [No description provided](0eeaa7b00067c130c4e32b7a642b439e) @ 2017-12-14T02:35:25Z
- [ DNS Performance Comparison: Google, Quad9, OpenDNS, Norton, CleanBrowsing, and Yandex https://medium.com/@nykolas.z/dns-performance-comparison-google-quad9-opendns-norton-cleanbrowsing-and-yandex-d62d24e38f98](0464d84536f1d97e322c3994cb5ea6e8) @ 2017-12-13T03:00:32Z
- [No description provided](1896eed3d1a3b28b85ce702535bd28bb) @ 2017-12-07T12:38:33Z
- [No description provided](6c38b8d1259de3c422c5f9e882761494) @ 2017-12-04T11:50:17Z
- [Take a nested list and yank out all of the data frames into a single depth=1 list.](4bda5de2888e02d3b7b053b202fcf02f) @ 2017-12-03T13:19:35Z
- [No description provided](ee6d72448b1df2e6d9223f901e922b4a) @ 2017-12-03T00:30:19Z
- [No description provided](a0487030f4f35ce4d075688e9d474ddb) @ 2017-11-29T10:57:22Z
- [No description provided](54e92c1233c260c611e33790b15d3d39) @ 2017-11-15T17:04:27Z
- [No description provided](6872c82e22cf5369639534388ea2599c) @ 2017-11-13T20:08:20Z
- [No description provided](92824182f41a8b841eab4480abc5a1d7) @ 2017-11-13T17:58:54Z
- [No description provided](3c2d9b31b5a7988f96145de9022f769e) @ 2017-10-30T10:15:52Z
- [No description provided](0c55e256f1873633e0b737b717722b38) @ 2017-10-25T19:10:41Z
- [find and extract emoji in R](e89eb173ae0333f50f94fe5086fedf8b) @ 2017-10-24T11:34:12Z
- [No description provided](f273131f8d2fe80237ac91f2f1909e97) @ 2017-10-23T20:26:12Z
- [use the magick device to make ggplots conform to twitter card or in-stream image optimal sizes, with or without "retina" resolution](0ce866666f124c23e3b42ac7f16f3d3b) @ 2017-10-21T14:39:04Z
- [composite a footer onto a ggplot plot – highly recommend doubling h & w, res = 144 and more customizing. this was a quick hack for someone.](6e353b8da97a1a1098ec053781b7bdac) @ 2017-10-20T12:57:21Z
- [ndjson record output sample for masscan](50f00ed7a6e627f0a897505ddd5a54d7) @ 2017-10-19T03:04:16Z
- [No description provided](31a1b4d36573d755cfbbd23d1d7093a7) @ 2017-10-18T14:16:03Z
- [No description provided](9e7125ae58a11a035b8330f497a6d98b) @ 2017-10-09T18:39:24Z
- [Rghaïf (North African flatbread)](5267f9e7479d21c16b1bbf1f05d23072) @ 2017-10-09T15:05:39Z
- [No description provided](f462277971b139a855651c99d26ac19e) @ 2017-09-27T14:08:14Z
- [No description provided](0ca210e2115665b7acbcc3d93187c21e) @ 2017-09-16T15:58:15Z
- [No description provided](944f9f5162ab40a9ed964ceb8992b93f) @ 2017-09-14T02:15:40Z
- [No description provided](1b380442c08dd3e36b8d71cc2a6a8b80) @ 2017-09-09T00:20:04Z
- [cost of hurricane harvey vis data — TSV here: https://static01.nyt.com/newsgraphics/2017/08/29/expensive-storms/79088630ae1af934d7840e104a0e3f1e8a6c7bf1/data-2.tsv](968f1df4ae5afa793141ce0290bdcfd9) @ 2017-09-05T13:57:15Z
- [Data from https://www.washingtonpost.com/graphics/world/nuclear-tests/ (you'll need to reverse the projected points)](52fb5fe1e818baa27b26ccf224464cf7) @ 2017-09-03T20:00:55Z
- [No description provided](5e069f20b8cbaf72b85ac839e2098d47) @ 2017-09-01T12:54:04Z
- [No description provided](42f3727544fe80e87785cf6eb06f1d2e) @ 2017-09-01T12:52:43Z
- [No description provided](60ef30f1f3821ed65e13d5214636d2e1) @ 2017-08-31T17:14:01Z
- [for #d3Makeover challenge #2 — it's not perfect, but you don't have to scrape, now. they changed fields and formats after 2011, so there's more detail in the later years. Not all fields had valid values. Earlier years had no expository.](0be6d4f71f7caff4830dffabed50082a) @ 2017-08-31T11:02:32Z
- [No description provided](b475def1e5a111dbabe3a903c06c76a4) @ 2017-08-28T14:13:33Z
- [get video playback time in Shiny](3c6ddebcc614b3099043404964ca0db0) @ 2017-08-28T13:25:23Z
- [No description provided](818a3a1b94d75abf95e88a7c9cb2d51b) @ 2017-08-25T00:43:10Z
- [No description provided](638e5651c380522abaeadf763e1fda6e) @ 2017-08-24T19:39:51Z
- [No description provided](8b6ccae02e13d5e19119d111a1508084) @ 2017-08-21T15:20:36Z
- [No description provided](96b0c86aa7e123db688f281b2702ac61) @ 2017-08-02T18:45:46Z
- [No description provided](bc46955e6254d98413d8bdd7725d5e66) @ 2017-08-01T02:50:51Z
- [robots.txt user agent strings from June 2017 robots.txt Common Crawl](7bfbc4a3b4cadb3a32d2e1b72440bf6a) @ 2017-08-01T02:37:27Z
- [No description provided](f8f2a60ec81621ed4233037edb63de0f) @ 2017-07-25T19:03:18Z
- [No description provided](c9eddb30e6fb5038085eb30b7d94f556) @ 2017-07-25T16:13:24Z
- [No description provided](82a3f74b0b353a63a09061207e9f2725) @ 2017-07-23T15:06:48Z
- [No description provided](dad2219d7a7659a6519c253365a34c43) @ 2017-07-23T13:36:56Z
- [No description provided](e177a8486599aba266a4fbe21ecdb853) @ 2017-07-16T02:12:29Z
- [30-45m weeknight recipes; these are simplified versions of more involved ones I have but these are go-to ones when I forget what time it is ("OH IT IS 7PM ALREADY?!") or schedules are really bad. they can all be adjusted for food allergies, too.](3abdfd489dd03e4ab89ec9c8e8a67c2e) @ 2017-07-13T19:01:24Z
- [No description provided](fa1f646bfecdff8e9d66c2cf537c8e92) @ 2017-07-11T21:41:33Z
- [No description provided](e22a82d1fb499e47e6187590a7e6cf77) @ 2017-07-11T15:05:25Z
- [No description provided](c55a1f3d267a7279e95b402e4be41069) @ 2017-07-02T13:36:05Z
- [Rapid7 Country-level results for SMB 445 scan that include protocol interaction (i.e. guaranteed/confirmed running SMB and on a Windows box). Data cross-referenced with National Exposure 2017 scan results to get total in-use IPv4 count to derive overall % of exposed Windows SMB](c790b975713705fa9c676bc1557e20af) @ 2017-06-29T17:25:47Z
- [Rapid7 Project Sonar - Full SMB Scan - Windows only results - counts by country - 2017-06-21](bc268f2ba91972a39a5794127a426195) @ 2017-06-27T17:26:11Z
- [trump & obama's approval ratings from day 1 in office](a7310e1b64d0797401d01d0c6195bd8b) @ 2017-06-18T14:16:21Z
- [No description provided](97eab930e2cf0846a42a76be37ae735c) @ 2017-06-14T19:37:28Z
- [No description provided](ec971ab031966b7148adc3dc2301b0f9) @ 2017-05-26T22:57:59Z
- [No description provided](ef7ea127e6244655a1782e6c3697ce64) @ 2017-05-26T22:52:30Z
- [No description provided](8c5cc3522670a80c5f63beb630854a4e) @ 2017-05-23T19:43:30Z
- [ggplot python](f848892299cbb29bb08f8e1a45650499) @ 2017-05-22T21:26:50Z
- [R palette for http://lewisandquark.tumblr.com/post/160776374467/new-paint-colors-invented-by-neural-network](1082a9b28120a22f6a7cf9e8ecae2eef) @ 2017-05-21T12:45:11Z
- [No description provided](5aa67459155e92225b23884cf60d3271) @ 2017-05-13T11:40:22Z
- [No description provided](ce0ccb2b0b38b37193154eb549ba9629) @ 2017-05-03T19:35:34Z
- [No description provided](e03deb479aae8e8b276f2be484823eed) @ 2017-04-30T16:11:47Z
- [No description provided](e6882569659e015de631e0cee7eb1f96) @ 2017-04-27T03:50:28Z
- [No description provided](6f228eead691e72bd0272a25319ea9ee) @ 2017-04-25T02:26:40Z
- [No description provided](a2169a38c5d99180f048de7723fcee99) @ 2017-04-24T11:07:08Z
- [No description provided](a7abae85fdd35e0bfdbab40c019e5078) @ 2017-04-21T16:17:33Z
- [No description provided](fa13f8b746ad4b00abbb45ede499f6bc) @ 2017-04-21T15:07:02Z
- [No description provided](8998143f299e9d59b826b01225dfb594) @ 2017-04-20T09:54:01Z
- [No description provided](b91752d32a07ee8595db33babe735b6b) @ 2017-04-13T19:33:05Z
- [No description provided](e73c4130902e22bbbf863c692bc2461b) @ 2017-04-13T11:21:52Z
- [No description provided](53d5cfc51f0fad9f0853c839029ef597) @ 2017-04-08T15:35:09Z
- [No description provided](1927e57e56749c09b75c4ab9f744ab46) @ 2017-04-02T16:09:30Z
- [No description provided](1b7f72aaac8b1bb0c4c99b549eb4a467) @ 2017-04-02T13:48:35Z
- [No description provided](49185cd3234eac5a9386f82a393f041f) @ 2017-03-28T14:46:09Z
- [No description provided](e8f3d531c4aa53e1019f6ff446bab88a) @ 2017-03-24T17:13:17Z
- [No description provided](33fa4154866cb6f257f1d52f7bc1e795) @ 2017-03-21T17:14:17Z
- [No description provided](880f609aa084b56c7f0326a6ee1c4c59) @ 2017-03-19T17:36:59Z
- [No description provided](61b15e8d039c883d431950facf411866) @ 2017-03-06T13:50:30Z
- [No description provided](8c7862092681d06e6535ab38d03074bf) @ 2017-03-04T11:49:34Z
- [No description provided](b088f26e71e8d88c0cc499b0f0a21344) @ 2017-03-03T20:17:37Z
- [Still needs finishing work for but this is def better than https://twitter.com/thosjleeper/status/836957509546098690](8579c9a5df8becbddfdaabdee7cd3543) @ 2017-03-01T18:08:02Z
- [No description provided](3116dfb8ba73f6307e2c42a0d6fcbf23) @ 2017-02-26T12:32:21Z
- [No description provided](3486db6f19c36c8a6bd3b1078f3841ba) @ 2017-02-22T18:52:02Z
- [No description provided](ebbb1b500c12e899504c33a262d41586) @ 2017-02-21T23:03:37Z
- [No description provided](00421bf0ce13410ea2044d61b0ffccf7) @ 2017-02-19T14:20:26Z
- [No description provided](453db36d1d3fc01c0a72bcc6f6312845) @ 2017-02-16T03:02:39Z
- [Riff of https://www.chezvoila.com/blog/parallel in R (where it's equally as 'ugh' to do)](18f80032277611be4f7375ba9c70099b) @ 2017-02-15T15:52:51Z
- [No description provided](664dabf73182fdcbc059f1c3c1675674) @ 2017-02-10T17:00:25Z
- [No description provided](323c195d840bd585f2aade450ad81631) @ 2017-02-10T05:16:17Z
- [No description provided](8dcefcaca8f86e6f65bec1cbe72ac5af) @ 2017-02-01T16:08:49Z
- [No description provided](108260491d615c37b938f047f80f5aa0) @ 2017-01-27T14:54:09Z
- [igraph layout editing with mapedit](6ca8308923bf8029285ace82e27bca6f) @ 2017-01-26T11:50:28Z
- [No description provided](95d795bf2e65fbb946450e1046ed6752) @ 2017-01-26T04:43:45Z
- [No description provided](8d90c883abdff66bc349a02a81e71457) @ 2017-01-24T17:11:00Z
- [Use custom local image files as icons in a Shiny Dashboard value box](605e62c5bf6deadf304d80cf4b1f0239) @ 2017-01-23T20:06:42Z
- [No description provided](f5d8c01f19f0f544a91a582486faf66a) @ 2017-01-22T15:29:49Z
- [No description provided](333206d2eef8b39fbf1080d583430e29) @ 2017-01-19T21:00:04Z
- [No description provided](b86780fde8799d2cb90b8428aba7fe45) @ 2017-01-09T15:54:13Z
- [No description provided](41212850a167da234637d677cc0c8fa8) @ 2017-01-05T23:03:34Z
- [No description provided](5c5db47e379f54768d9ccfe13ba0eb6d) @ 2017-01-03T18:23:30Z
- [R JRI](ec02bdd32523f8e176f4bd4f62ec0541) @ 2016-12-30T13:00:36Z
- [decoding a polyline in Rcpp/C++11](01dfdd6f32e151330a8cc6f772574b9f) @ 2016-12-20T18:25:08Z
- [No description provided](9bd23ec850c66f4b48811315079fadef) @ 2016-12-09T16:55:04Z
- [R version of http://stephanieevergreen.com/two-alternatives-to-using-a-second-y-axis/](859018befbfb8de25328fffba8b27ef9) @ 2016-12-07T17:30:47Z
- [No description provided](6e9cbfdffcce7b3b0e5d5fcf373660f5) @ 2016-12-03T22:59:43Z
- [No description provided](3a4d920ae45ca15d2eae190f457a6806) @ 2016-11-29T15:02:20Z
- [No description provided](ce5fe39695a99424eeea1fbab9282d74) @ 2016-11-19T14:05:30Z
- [No description provided](bcdee1bc9bebab5bf521b995ebd68b16) @ 2016-11-16T04:56:38Z
- [No description provided](bbbacd13b043ca16257208a1357994dc) @ 2016-10-20T02:40:04Z
- [How to re-create Ann K Emery's "Double Y axis" fix @ http://annkemery.com/doubley/ in R + ggplot2](ff60ef4106a2676cc91a40cb40c6e288) @ 2016-10-19T19:04:42Z
- [No description provided](3cf8e0db622a7ea21f7f79eea285798d) @ 2016-09-28T00:47:38Z
- [No description provided](929f6b8ebf8e878ce5a323e6a4bfce64) @ 2016-09-26T17:19:18Z
- [No description provided](900c1ae5ac1741422af4e03e6678abc5) @ 2016-09-19T14:35:44Z
- [No description provided](324a1e263904c3c43f48c62744fa3643) @ 2016-09-18T18:18:31Z
- [No description provided](d0f635fe94ce25b5653906297e60d787) @ 2016-09-15T15:21:27Z
- [No description provided](cada8c70e6027294f06fe9a9afee4fc0) @ 2016-09-12T19:18:50Z
- [No description provided](2e64547e7a0e5ee13d62389fb7e61282) @ 2016-09-07T21:48:06Z
- [No description provided](586ceec0f72ee9a6a932a759b744c6a8) @ 2016-07-28T15:02:39Z
- [Context: https://rud.is/b/2016/07/27/u-s-drought-animations-with-the-witchs-brew-purrr-broom-magick/](c8e001168e1ed7369c637be0589ed768) @ 2016-07-28T14:35:53Z
- [No description provided](5d461857b40e2ad5558d769f2de00174) @ 2016-07-20T13:04:33Z
- [No description provided](8bb810fb33646717ff4a1091a8c9fe91) @ 2016-07-19T16:20:22Z
- [No description provided](1d81926d90e378a6ef55ab33faae196a) @ 2016-07-16T10:58:35Z
- [No description provided](85979722178dde91b2680dad6a487963) @ 2016-07-11T20:02:35Z
- [No description provided](afe37dc84fcaf784522b443c0693ee23) @ 2016-07-06T14:00:46Z
- [smaller shapfiles](539d02f25b8f33d80edf7bbead627b05) @ 2016-07-05T01:05:39Z
- [No description provided](f388bdc27b5ba987d82fcf45f2080bf9) @ 2016-06-29T01:29:12Z
- [No description provided](162aebd36a8d76a563ef8a1ecda4414a) @ 2016-06-24T15:22:08Z
- [Overlapping bars in ggplot2 : ref http://stephanieevergreen.com/overlapping-bars/](035f998517de2384e9962cff7df874bd) @ 2016-06-23T13:27:44Z
- [No description provided](470a42a35df7eb5277f2f5088fad9835) @ 2016-06-22T17:29:30Z
- [CRAN pkgs with github `URL:`s](3468316797719c67edb453380aa53a00) @ 2016-06-21T15:02:01Z
- [priestly timeline for FT blog post](f12101f8ef1657f9e1457e0dde1602f8) @ 2016-06-16T13:49:31Z
- [No description provided](a1e35d04d93be1496e246633701c54e1) @ 2016-06-12T17:12:43Z
- [parallel, parameterized knitr reports](17bc21af55392f23f012f57bb2fda51c) @ 2016-06-04T17:50:25Z
- [No description provided](b382d30e07083746c69854343b54e670) @ 2016-06-03T13:10:29Z
- [Commutes 2012 - Oldskool style](301f0c9732205a3dd4670f8e3d112df8) @ 2016-05-31T02:11:47Z
- [No description provided](f11429b746b838fd382e1132492232c7) @ 2016-05-24T11:32:09Z
- [Visualizing Distributions](0697c2d6dd3ceee7104d5fc306a725e4) @ 2016-05-21T10:53:20Z
- [species radar](a91dfc85fc5838892d4a50786de54710) @ 2016-05-17T11:36:23Z
- [R code to reproduce the awesome visualization of global temperature change from Ed Hawkins at http://www.climate-lab-book.ac.uk/2016/spiralling-global-temperatures/ using R and ggplot2 (with the animations package)](1849717a390142a7bf76aa5d6ea80577) @ 2016-05-14T11:47:55Z
- [No description provided](f9c3bbc561e824219954ab5eecc12f2b) @ 2016-05-13T22:55:37Z
- [No description provided](ec35ebb32c3cf0aba95f7bad28df1e98) @ 2016-05-10T20:09:43Z
- [Reproduction of the scatterplot on: http://www.nytimes.com/2016/05/10/upshot/where-democrats-like-hillary-clinton-the-least-besides-vermont.html?_r=0](4632dc9cefc7566b412ec9945a6c4922) @ 2016-05-09T16:48:38Z
- [No description provided](57596ef68695d34e422c2e8167322cdb) @ 2016-05-07T23:06:02Z
- [No description provided](e60f24ada36f6c58d69c3f3d7cb2f143) @ 2016-05-05T00:40:23Z
- [No description provided](dbb3bbf8f5750c26ef38ec22e47c3686) @ 2016-05-03T01:29:46Z
- [No description provided](658140e9b7e6928c9db931b6be91257b) @ 2016-05-03T01:15:12Z
- [No description provided](f9e55fd50ac058b6ad20c01594f2ee9e) @ 2016-05-03T00:35:56Z
- [No description provided](910af8ddc6371572aa4414b77ae86c6a) @ 2016-05-02T00:57:17Z
- [No description provided](65db545f5fa32b7916e189e430df2aaf) @ 2016-04-29T10:30:58Z
- [French towns, surnames and suffixes](0fd37cf3825fc8e3eddf042a4443d1dc) @ 2016-04-29T10:03:24Z
- [No description provided](ba8bee6275e91698e679e7e61665493b) @ 2016-04-26T01:02:02Z
- [No description provided](a73e88de97d41149decb75b1f72cf268) @ 2016-04-23T12:15:11Z
- [No description provided](596636d59d9bc6301a239fb8b13bd305) @ 2016-04-21T00:43:01Z
- [Logistic regression tutorial code. Full article available at http://datascienceplus.com/perform-logistic-regression-in-r/](4c21c5ddad2867f6a00867cd72ad95a5) @ 2016-04-17T16:59:18Z
- [No description provided](07bd552711ad699e437448c4b069ba88) @ 2016-04-12T12:41:10Z
- [No description provided](993f739d984b445fbdc785e675053234) @ 2016-04-12T00:45:47Z
- [quickstart SQL db environs](86dd5d55cf1f93f71738d441ba3dac5d) @ 2016-04-11T15:41:48Z
- [No description provided](6bc42e9afdd00f8f65400670b24e151f) @ 2016-03-30T23:56:44Z
- [No description provided](3730d766d0fd6e4bdbac) @ 2016-03-29T12:24:17Z
- [Nuclear Explosions in D3](25128ed9893cf0d1d08c) @ 2016-03-28T02:33:37Z
- [No description provided](626b1b3a3f4bf744b811) @ 2016-03-26T18:54:51Z
- [No description provided](d91145aa8d7380ff73f8) @ 2016-03-26T14:25:26Z
- [No description provided](d5c22db162ecc19b0fbc) @ 2016-03-26T12:47:23Z
- [No description provided](a61991ebd8f4f49ce739) @ 2016-03-25T01:54:14Z
- [No description provided](02687116d7697a9fe007) @ 2016-03-24T23:04:32Z
- [No description provided](7c42b1659d476aebe091) @ 2016-03-23T22:51:12Z
- [2015 Fortune 1000 List with industry & website](ae574201af3de035c684) @ 2016-03-23T16:02:12Z
- [Nuclear Explosions](6ebc8476a44ca2ffc2d4) @ 2016-03-23T13:49:13Z
- [Calling R's message() from C in a package without using Rcpp](cd6cc2975155852e22d5) @ 2016-03-22T12:51:44Z
- [No description provided](e4a0cc2b3fc958a297ea) @ 2016-03-22T04:13:45Z
- [No description provided](3daa349bcefc63041ff7) @ 2016-03-21T12:34:40Z
- [apple watch ggplot](80685ea0cbdf9b4afbad) @ 2016-03-20T16:09:55Z
- [No description provided](99608f5b87787c3b155d) @ 2016-03-19T11:27:01Z
- [the data behind http://www.usatoday.com/story/news/2016/03/11/nearly-2000-water-systems-fail-lead-tests/81220466/](47c7a8d92ba066a8add7) @ 2016-03-18T23:45:45Z
- [Stacking the deck against treemaps - http://rud.is/b/2016/03/18/stacking-the-deck-against-treemaps/](8094644863ba7ce3959c) @ 2016-03-18T18:19:30Z
- [Supreme Annotations - moar splainin here: http://rud.is/b/2016/03/16/supreme-annotations/ - NOTE: this requires the github version of ggplot2](283850725519e502e70c) @ 2016-03-16T00:59:03Z
- [No description provided](84c65834444c24192ebd) @ 2016-03-15T00:46:03Z
- [No description provided](29e6ab86507d20beac01) @ 2016-03-15T00:44:48Z
- [No description provided](c8c30bf0b94d5d0e157e) @ 2016-03-14T21:58:29Z
- [No description provided](33c32e369daba38f9ef9) @ 2016-03-10T16:57:48Z
- [No description provided](936fce53734251a9df52) @ 2016-03-10T11:45:59Z
- [No description provided](96f7ff68e4fd6ffe154f) @ 2016-03-10T01:27:44Z
- [No description provided](f7436224d6d32300e8ae) @ 2016-03-08T12:07:35Z
- [No description provided](7a503faf51b74d15778a) @ 2016-03-07T20:20:23Z
- [No description provided](73f4ad957314b5a847c1) @ 2016-03-07T15:25:02Z
- [No description provided](25a53e2fcaee2aafa908) @ 2016-03-07T13:57:14Z
- [No description provided](b650014f065923def9f9) @ 2016-03-06T20:45:54Z
- [hack-ish way to get all the code examples from your package](80393a8c88c84b84f3ba) @ 2016-03-06T16:31:07Z
- [AUS BOM R thing](9f09102fc3da283b707c) @ 2016-03-03T14:18:26Z
- [via https://github.com/iingemar/pokegraphs](8e31d43afc7b0179e4df) @ 2016-03-02T15:44:33Z
- [time series detail for the asn safe search thing](246cdaf81d0ad10e7c86) @ 2016-03-01T18:19:07Z
- [Google Safe Search AS scanning data "API" wrapper](4a4ac735062b49ef8e1f) @ 2016-03-01T16:53:48Z
- [No description provided](60672a352ded71e81643) @ 2016-03-01T03:00:51Z
- [No description provided](7f9c4de02c08ddef4ac6) @ 2016-02-28T17:00:35Z
- [No description provided](115997024fb0ec06612f) @ 2016-02-25T13:33:07Z
- [No description provided](df646876b799b1dfd313) @ 2016-02-24T12:42:07Z
- [benchmarking R line readers](c059d1fbda689aab111a) @ 2016-02-23T12:20:27Z
- [No description provided](42480de78a3ee1ed39bb) @ 2016-02-18T04:59:18Z
- [No description provided](fcf6d46d98f07a247d83) @ 2016-02-17T10:42:43Z
- [No description provided](6251eb302d6945da682b) @ 2016-02-17T02:44:12Z
- [Code for this post: ](696b124d0190bba15817) @ 2016-02-16T04:33:34Z
- [No description provided](c10dae0230b11f9d1e82) @ 2016-02-15T23:59:16Z
- [fanduel densities](3c0200feb8aca04ae767) @ 2016-02-14T19:56:49Z
- [New R Logo as WKT](07d0ccf14c2ff109f55a) @ 2016-02-11T11:58:42Z
- [Markers and request from - http://anasim.iet.unipi.it/moniqa/#home](dae86a8688058e4a5e46) @ 2016-02-09T21:29:33Z
- [scraped PDF of latest (http://www.michigan.gov/documents/flintwater/January_29_2016_Test_Results_Flint_Sorted_by_Address_512511_7.pdf) water tests](a16d326ac3582eacfb08) @ 2016-02-07T02:13:57Z

15375ec7a873d17ea5e2/CHANGELOG.md
#### 2015-07-29 

- Added instructions for loading job with `launchctl`
- Fixed issue in `vol=${atch##* }`

6503449/wknds.R
# need "chron" package for is.weekend()
require(chron)

# generate a sequence of dates
m.days <- seq.Date(from=as.Date("2013-08-01"), to=as.Date("2013-08-31"), by="1 day")

# if a day in the sequence is a weekend, return it
wknds <- ldply(m.days, function(m.day) {
  if (is.weekend(m.day)) m.day
})

# makes the following:
#           V1
# 1 2013-08-03
# 2 2013-08-04
# 3 2013-08-10
# 4 2013-08-11
# 5 2013-08-17
# 6 2013-08-18
# 7 2013-08-24
# 8 2013-08-25
# 9 2013-08-31

# now make a data frame with the weekend dates and which dow it is
wknds.df <- data.frame(d=wknds$V1, dow=weekdays(wknds$V1))

#            d      dow
# 1 2013-08-03 Saturday
# 2 2013-08-04   Sunday
# 3 2013-08-10 Saturday
# 4 2013-08-11   Sunday
# 5 2013-08-17 Saturday
# 6 2013-08-18   Sunday
# 7 2013-08-24 Saturday
# 8 2013-08-25   Sunday
# 9 2013-08-31 Saturday
